src/libcore/tests/cfg_if_override.rs

```diff
@@ -0,0 +1,18 @@
+//! Test that overriding cfg_if with our own cfg_if macro does not break
+//! anything.
+#![feature(cfg_if)]
```

Comment:
This directive shouldn't be required, right?

Reply:
I think so. It should only be required if the `cfg_if!` from `core` is used, but that shouldn't happen here.